File: Sample/Instabug Demo/Support/UIAlertAction+DefaultAction.m
//
//  UIAlertAction+DefaultAction.m
//  Instabug Demo
//
//  Copyright © 2016 Instabug. All rights reserved.
//

#import "UIAlertAction+DefaultAction.h"

@implementation UIAlertAction (DefaultAction)

+ (instancetype)ibg_actionWithTitle:(NSString *)title handler:(void (^)(UIAlertAction *action))handler {
    UIAlertAction *action = [UIAlertAction actionWithTitle:title
                                                     style:UIAlertActionStyleDefault
                                                   handler:handler];
    
    return action;
}

@end

File: Sample/Instabug Demo/Support/UIAlertAction+DefaultAction.swift
//  UIAlertAction+DefaultAction.m
//  Instabug Demo
//
//  Copyright © 2016 Instabug. All rights reserved.
//

import UIKit

extension UIAlertAction {
    static func ibg_action(withTitle title: String?,
                           handler: @escaping (_ action: UIAlertAction?) -> Void) -> UIAlertAction {
        let action = UIAlertAction(title: title, style: .default, handler: handler)
        return action
    }
}
